runtime-sdk/src/modules/rofl/app/mod.rs

```diff
@@ -31,14 +31,18 @@ pub use env::Environment;
 #[allow(unused_variables)]
 #[async_trait]
 pub trait App: Send + Sync + 'static {
-    /// Runtime to attach the application to.
-    ///
-    /// The runtime must have the ROFL module enabled so that the application can register to it.
-    type AttachTo: Runtime;
+    /// ROFL application version.
+    const VERSION: version::Version;
 
     /// Identifier of the application (used for registrations).
     fn id() -> AppId;
 
+    /// Return the consensus layer trust root for this runtime; if `None`, consensus layer integrity
+    /// verification will not be performed.
```

Comment:
This looks dangerous. Should we rather force apps to implement this, even if they only have to return `None`, so that they explicitly ignore verification?

Reply:
Good point, let me fix this, the PR already got merged.

Reply:
Sorry, didn't see that you have auto-merge enabled. If needed, we can change this in another PR.

Reply:
#1885